### ISSUE TITLE
Permission denied error when attempting to install local AI plugin server

### ISSUE THREAD

**Issue by josephlevin** [Acly/krita-ai-diffusion#68]
EDIT: I copied the entirety of the subfolder containing the python, etc. installation files from the laptop to the desktop PC, and I got the server to work. Not the ideal solution, or even a good fix, maybe it'll help someone else to know this is possible.

Example of the error from the client.log:
2023-11-19 12:44:32,992 INFO Downloading https://www.python.org/ftp/python/3.10.11/python-3.10.11-embed-amd64.zip to C:\Users\josep\AppData\Roaming\krita\pykrita\ai_diffusion\.server\.cache\python-3.10.11-embed-amd64.zip
2023-11-19 12:44:33,108 ERROR Failed to download https://www.python.org/ftp/python/3.10.11/python-3.10.11-embed-amd64.zip: Permission denied

I just installed Krita and the AI plugin on my laptop, so manually placed the above ZIP file into the server cache folder, as a test. 

To whit, the next file that was attempted to download received the same error:
2023-11-19 12:57:08,069 INFO Downloading https://bootstrap.pypa.io/get-pip.py to C:\Users\josep\AppData\Roaming\krita\pykrita\ai_diffusion\.server\python\get-pip.py
2023-11-19 12:57:08,255 ERROR Failed to download https://bootstrap.pypa.io/get-pip.py: Permission denied

I am guessing there is something off with the folder permissions, but it has full control for each account slated to use it.

Any suggestions?

(PS: Both machines are almost identically running the same software, with Windows 10 22H2 as the OS, if that helps.

Thanks!

**Comment by Acly** [comment]
It's curious, I don't have any good ideas either.
Did you try installing into a different path (maybe not in the user directory)? It can be anywhere.
Inexplicable permission denied errors are sometimes casue by over-eager anti-virus, but it's a long shot.

**Comment by josephlevin** [comment]
I used the default installation folder. I also disabled my firewall and antivirus temporarily. I did get lucky in that copying over the newly installed AI/python related files from one computer to the other allowed the computer with the permission issue to continue with the installation and then successfully generate content within Krita. 

It's not ideal not knowing what was at issue, but I'm counting this as a win.

Thank you very much for providing the means to use Stable Diffusion within Krita!
